wd_util.c

```diff
@@ -1051,7 +1096,13 @@ static int wd_init_async_polling_thread_per_numa(struct wd_env_config *config,
 	head = task_queue;
 	config_numa->async_task_queue_array = (void *)head;
 
-	for (i = 0; i < config_numa->async_poll_num; task_queue++, i++) {
+	if (config_numa->async_poll_num > config_numa->async_ctx_num) {
+		n = config_numa->async_ctx_num;
+		WD_ERR("Create %ld async polling threads since ctx number is "
+		       "limited.\n", config_numa->async_poll_num);
```

Comment:
here do you want to create poll thread which is the number of async_ctx_num when config_numa->async_poll_num > config_numa->async_ctx_num, if so, there is a wrong print